src/configuration/app-containers.md

```diff
@@ -51,59 +52,59 @@ web:
 # The size of the persistent disk of the application (in MB).
 disk: 2048
 
-# The 'mounts' describe writable, persistent filesystem mounts in the application. The keys are
-# directory paths, relative to the application root. The values are strings such as
-# 'shared:files/NAME', where NAME is just a unique name for the mount.
+# The 'mounts' describe writable, persistent filesystem mounts in the application.
+# The keys are directory paths relative to the application root. The values are
+# strings such as 'shared:files/NAME' where NAME is just a unique name for the mount.
 mounts:
     '/web/files': 'shared:files/web-files'
 
-# The hooks that will be performed when the package is deployed.
+# The hooks that will be triggered when the package is deployed.
 hooks:
-    # Build hooks can modify the application files on disk, but not access any services like databases.
+    # Build hooks can modify the application files on disk but not access any services like databases.
     build: |
       rm web/app_dev.php
-    # Deploy hooks can access services, but the file system is now read-only.
+    # Deploy hooks can access services but the file system is now read-only.
     deploy: |
       app/console --env=prod cache:clear
 ```
 
 > **Note**
 > This configuration file is specific to one application. If you have multiple
-> applications inside your Git repository (i.e. a RESTful
-> web service and a front-end, or a main web site and a blog), you need
-> one `.platform.app.yaml` at the root of each application. See the [Multi-app](/configuration/app/multi-app.md) documentation.
+> applications inside your Git repository (i.e. a RESTful web service and a
+> front-end or a main web site and a blog), you need one `.platform.app.yaml`
+> at the root of each application. See the [Multi-app](/configuration/app/multi-app.md) documentation.
```

Comment:
I'm unclear what's changing here.  However, I'd argue the comma within the parenthetical is needed as otherwise we're left with a string of clauses linked by ands and ors, and left with order of operations to determine what goes where.  English's order of operations in such cases is highly inconsistent. :smile: